docs/requirements.txt

```diff
@@ -1,27 +1,2 @@
-alabaster==0.7.13
-Babel==2.11.0
-beautifulsoup4==4.11.1
-certifi==2022.12.7
-charset-normalizer==3.0.1
-docutils==0.19
-idna==3.4
-imagesize==1.4.1
-Jinja2==3.1.2
-MarkupSafe==2.1.2
-maturin==0.14.10
-packaging==23.0
-Pygments==2.14.0
-pytz==2022.7.1
-requests==2.28.2
-shibuya==2023.6.30
-snowballstemmer==2.2.0
-soupsieve==2.3.2.post1
-Sphinx==6.1.3
-sphinx-basic-ng==1.0.0b1
-sphinxcontrib-applehelp==1.0.4
-sphinxcontrib-devhelp==1.0.2
-sphinxcontrib-htmlhelp==2.0.0
-sphinxcontrib-jsmath==1.0.1
-sphinxcontrib-qthelp==1.0.3
-sphinxcontrib-serializinghtml==1.1.5
-urllib3==1.26.14
+shibuya
+sphinx
```

Comment:
Using unpinned requirements for docs builds is usually fine - I haven't had any issues with even my larger projects.